src/popup/components/Truncate.vue

```diff
@@ -125,23 +122,18 @@ export default defineComponent({
       position: relative;
       display: inline-block;
       min-width: fit-content;
-      transition: opacity 0.5s;
-      opacity: 0;
-
-      &.ready {
-        opacity: 1;
-      }
     }
 
     &.scrollable {
       padding-right: 5px;
       border-radius: 2px;
+      margin-left: -2px;
       mask-image:
         linear-gradient(
           90deg,
           rgba(black, 0) 0%,
-          rgba(black, 1) 5%,
-          rgba(black, 1) 95%,
+          rgba(black, 1) 2px,
+          rgba(black, 1) calc(100% - 2px),
```

Comment:
Here we are defining the size of the gradient that covers the beginning and the end of the string. In my opinion 2px is too small.